src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -85,4 +92,25 @@ public void deleteTaskFromModule(Module targetModule) throws ModHappyException {
         TaskList taskList = targetModule.getTaskList();
         result = String.format(DELETE_MESSAGE, taskList.removeTask(taskIndex));
     }
+
+    /**
+     * Gets confirmation from user to delete given module.
+     *
+     * @param module Module to be deleted.
+     * @return Returns true if user input is "yes", false if "no".
+     * @throws ParseException Throws an exception if user input is not "yes" or "no".
+     */
+    public Boolean getUserConfirmation(Module module) throws ParseException {
+        Scanner scanner = new Scanner(System.in);
+        String prompt = String.format(DELETE_CONFIRMATION, module);
+        System.out.println(prompt);
+        String userConfirmation = scanner.nextLine().toLowerCase();
+        if (userConfirmation.equals("yes")) {
+            return true;
+        }
+        if (userConfirmation.equals("no")) {
+            return false;
+        }
+        throw new ParseException();
```

Comment:
In my opinion, I think a loop here instead of the exception might be better as discussed

Reply:
That's a good idea, with a loop the user won't have to retype the command if they mistype yes or no.